gitlab/v4/objects/projects.py

```diff
@@ -894,6 +894,65 @@ def remote_import(
             "/projects/remote-import", post_data=data, **kwargs
         )
 
+    def remote_import_s3(
+        self,
+        region: str,
+        bucket_name: str,
+        file_key: str,
+        access_key_id: str,
+        secret_access_key: str,
+        path: str,
+        name: Optional[str] = None,
+        namespace: Optional[str] = None,
+        overwrite: bool = False,
+        override_params: Optional[Dict[str, Any]] = None,
```

Comment:
Should we maybe have the path at the top here? Will be following the upstream docs for required attributes closer, visually.

```suggestion
        path: str,
        region: str,
        bucket_name: str,
        file_key: str,
        access_key_id: str,
        secret_access_key: str,
        name: Optional[str] = None,
        namespace: Optional[str] = None,
        overwrite: bool = False,
        override_params: Optional[Dict[str, Any]] = None,
```

Reply:
Oh right, will do. I was following https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/api/project_import.rb#L161 
Actually the docs do not mention the parameters `overrid_params` and `overwrite` in the list of params. In description they do mention `override_params`. Was confused, hence checked the code out :). But people are going to read from spec, so this is the right way. Thanks!

Reply:
but according to this, shouldn't name, namespace be at the top? 
https://docs.gitlab.com/ee/api/project_import_export.html#import-a-file-from-aws-s3

Reply:
lol sorry they are optional

Reply:
Done, updated at other places accordingly. Pls resolve if correct